lib/ansible/modules/cloud/amazon/rds_instance.py

```diff
@@ -1153,7 +1155,7 @@ def main():
     ]
     mutually_exclusive = [
         ('s3_bucket_name', 'source_db_instance_identifier', 'snapshot_identifier'),
-        ('use_latest_restorable_time', 'restore_to_time'),
+        ('use_latest_restorable_time', 'restore_time'),
```

Comment:
Can this cause existing roles/playbooks to suddenly fail?

Reply:
Currently AWS itself would throw an error rather than Ansible.

Reply:
Ok, good :)